### PR TITLE
added support for 2.1.0

### PR DESCRIPTION
**Pull request by alkimake**
There were minor changes for support to Taiga:2.1.0

### CODE REVIEW

File: taiga-front-dist/configure [hunk 1]
```diff
@@ -22,16 +22,22 @@ API=${API:-/api/v1/}
...
-  "privacyPolicyUrl": null,
-  "termsOfServiceUrl": null,
-  "maxUploadFileSize": null,
-  "gitHubClientId": null,
-  "contribPlugins": [ ]
+    "api": "${API}",
+    "eventsUrl": null,
+    "eventsMaxMissedHeartbeats": 5,
+    "eventsHeartbeatIntervalTime": 60000,
+    "debug": true,
```
Comment (htdvisser) [inline taiga-front-dist/configure]: Why add debug:true?

File: taiga-back/configure [hunk 2]
```diff
@@ -7,10 +7,12 @@ POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-$POSTGRES_ENV_POSTGRES_PASSWORD}
 
 FRONT_SCHEME=${FRONT_SCHEME:-http}
 FRONT_DOMAIN=${FRONT_DOMAIN:-$HOSTNAME}
+FRONT_NAME=${FRONT_NAME:-front}
 MEDIA_URL=${MEDIA_URL:-$FRONT_SCHEME://$FRONT_DOMAIN/media/}
 STATIC_URL=${STATIC_URL:-$FRONT_SCHEME://$FRONT_DOMAIN/static/}
 API_SCHEME=${API_SCHEME:-http}
 API_DOMAIN=${API_DOMAIN:-$HOSTNAME}
+API_DOMAIN=${API_NAME:-api}
```
Comment (htdvisser) [inline taiga-back/configure]: I guess this should be `API_NAME=${API_NAME:-api}`

**Comment by htdvisser** [comment]
Could someone else please verify that this change is correct?

File: taiga-front-dist/configure (hunk 1 above)
Comment (alkimake) [inline taiga-front-dist/configure]: Ohhh, i am so sorry, i pushed unfinished commits... I will fix problems and send you. My bad.

**Pull request merged**